src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/RemoveValidator.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.RemoveDto;
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationMessages;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.setErrorMsgToLocation;
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationUtils.getQualifiedFieldName;
+
+@Component
+public class RemoveValidator {
+
+    public Errors validate(RemoveDto removeDto, Errors errors, String loggingContext) {
+        if (removeDto.getIsNotProprietorOfLand() != null && removeDto.getIsNotProprietorOfLand() == false) {
```

Comment:
Thinking about this, logic should perhaps be:

`removeDto.getIsNotProprietorOfLand() == null || removeDto.getIsNotProprietorOfLand() == false`

Reply:
i agree

Reply:
Now not sure. The above holds true for _full_ validation within the API but perhaps not for _partial_ validation (where field may well be not present/null, at least early on in the journey). @HatiDev we will have to test this when you get the validator code running in Tilt.